exec/totemknet.c

```diff
@@ -906,12 +914,22 @@ void totemknet_configure_log_level()
 			knet_log_mode = KNET_LOG_DEBUG;
 			break;
 		case LOGSYS_DEBUG_TRACE:
+#ifdef KNET_LOG_TRACE
+			knet_log_mode = KNET_LOG_TRACE;
+#else
 			knet_log_mode = KNET_LOG_DEBUG;
+#endif
 			break;
 	}
 	log_printf (LOGSYS_LEVEL_DEBUG, "totemknet setting log level %s", knet_log_get_loglevel_name(knet_log_mode));
+	err = 0;
 	for (s = 0; s<KNET_MAX_SUBSYSTEMS; s++) {
-		knet_log_set_loglevel(global_instance->knet_handle, s, knet_log_mode);
+		err = knet_log_set_loglevel(global_instance->knet_handle, s, knet_log_mode);
+	}
+
+	/* If one fails, they all fail. no point in issuing KNET_MAX_SUBSYSTEMS errors */
+	if (err) {
+		log_printf (LOGSYS_LEVEL_INFO, "totemknet failed to set log level: %s", strerror(errno));
```

Comment:
Can we change it to higher level than info (error/warning/notice)? I think info level is too low.